app/views/alchemy/admin/pictures/_picture_to_assign.html.erb

```diff
@@ -18,7 +12,7 @@
     action_url,
     remote: true,
     onclick: '$(self).attr("href", "#").off("click"); return false',
-    method: action_method,
+    method: @action_method,
```

Comment:
It's generally a bit terrible, but if you just put it here and delete all the other stuff, then the code reader won't have to trace back all the way to the controller. :)